tests/python/PyDocStyleBearTest.py

```diff
@@ -23,7 +23,37 @@ def hello():
 
 '''.splitlines(keepends=True)
 
-PyDocStyleBearTest = verify_local_bear(PyDocStyleBear,
-                                       valid_files=(good_file,),
-                                       invalid_files=(bad_file,),
-                                       tempfile_kwargs={"suffix": ".py"})
+PyDocStyleBearTest = verify_local_bear(
+    PyDocStyleBear,
+    valid_files=(good_file,),
+    invalid_files=(bad_file,),
+    tempfile_kwargs={"suffix": ".py"})
+
+PyDocStyleBearIgnoreArgTest = verify_local_bear(
+    PyDocStyleBear,
+    valid_files=(good_file,),
+    invalid_files=(bad_file,),
+    settings={'pydocstyle_ignore': 'D400'},
+    tempfile_kwargs={"suffix": ".py"})
+
+PyDocStyleBearSelectArgTest = verify_local_bear(
+    PyDocStyleBear,
+    valid_files=(good_file,),
+    invalid_files=(bad_file,),
+    settings={'pydocstyle_select': 'D200'},
+    tempfile_kwargs={"suffix": ".py"})
```

Comment:
your tests don't really validate that it works, to do that you'd need to e.g. select only routines that don't yield a result so the bad file gets good